lib/rialto/etl/configs/wos_to_sparql_statements.rb

```diff
@@ -123,13 +119,21 @@ def lookup_address(addresses, addr_id)
       person[VCARD['hasAddress'].to_s] = address_vcard
     end
 
-    {
+    # If there is an organization, add a position for the person
+    if address && address.key?('organization')
+      person['#organization'] = Rialto::Etl::Transformers::People.construct_position(org_name: address['organization'],
+                                                                                     person_id: person_id)
+    end
+
+    authorship = {
```

Comment:
There's no need to make this assignment, if this is the last statement, this will be returned.